src/sentry/models/group.py

```diff
@@ -149,6 +149,21 @@ def from_event_id(self, project, event_id):
 
         return Group.objects.get(id=group_id)
 
+    def filter_by_event_id(self, project_ids, event_id):
+        from sentry.models import EventMapping, Event
+        group_ids = set()
+        # see above for explanation as to why we're
+        # looking at both Event and EventMapping
+        for model in Event, EventMapping:
+            group_ids.update(
+                model.objects.filter(
+                    project_id__in=project_ids,
+                    event_id=event_id,
```

Comment:
We should probably filter out null `group_ids` here, can probably just be done in query?